x-pack/plugins/maps/public/actions/map_actions.ts

```diff
@@ -384,8 +386,17 @@ export function addNewFeatureToIndex(geometry: Geometry | Position[]) {
     if (!layer || !(layer instanceof VectorLayer)) {
       return;
     }
-    await layer.addFeature(geometry);
-    await dispatch(syncDataForLayer(layer, true));
+
+    try {
+      await layer.addFeature(geometry);
+      await dispatch(syncDataForLayer(layer, true));
+    } catch (e) {
+      getToasts().addError(e, {
+        title: i18n.translate('xpack.maps.mapActions.addFeatureError', {
+          defaultMessage: `Error adding feature to index`,
```

Comment:
We typically don't use the word "Error" in our messages.  Instead, we use this format:

Unable to add feature to index